backend/remote/backend.go

```diff
@@ -497,6 +521,15 @@ func generalError(msg string, err error) error {
 	}
 }
 
+const generalErr = `
+%s: %v
+
+The "remote" backend encountered an unexpected error while communicating
+with remote backend. In some cases this could be caused by a network
```

Comment:
This first sentence is a little awkward, can we do something to get rid of the double "remote"?

Reply:
I see what you mean. I'll improve that a bit before merging the PR.